sdk/include/ndk/rtltypes.h

```diff
@@ -847,7 +847,6 @@ typedef struct _UNICODE_PREFIX_TABLE
     PUNICODE_PREFIX_TABLE_ENTRY LastNextEntry;
 } UNICODE_PREFIX_TABLE, *PUNICODE_PREFIX_TABLE;
 
-#ifdef NTOS_MODE_USER
```

Comment:
One question: without this define, the Pfx structures become available in both user and kernel mode?

Reply:
"To figure out what this macro does is left as an exercise to the reader."